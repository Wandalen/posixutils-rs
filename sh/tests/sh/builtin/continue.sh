for x in a b c
do
  continue
  echo $x
done

for x in a b c
do
  echo correct && continue && echo wrong
  echo $x
done

for x in a b c
do
  continue 1
  echo $x
done

for x in a b c
do
  continue 10
  echo $x
done

for x in a b c
do
  echo $x
  for y in d e f
  do
    echo $y
    continue
  done
done

for x in a b c
do
  echo $x
  for y in d e f
  do
    echo $y
    continue 1
  done
done

for x in a b c
do
  echo $x
  for y in d e f
  do
    echo $y
    continue 2
  done
done

var='continue'
while { echo 'in condition'; test $var = 'continue'; }
do
  var='stop'
  continue
  echo wrong
done

var='stop'
until { echo 'in condition'; test $var = 'continue'; }
do
  var='continue'
  continue
  echo wrong
done

for x in a b c; do
  continue --
  echo wrong
done

for x in a b c; do
  continue -- 1
  echo wrong
done