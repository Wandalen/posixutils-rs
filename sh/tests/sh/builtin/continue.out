correct
correct
correct
a
d
e
f
b
d
e
f
c
d
e
f
a
d
e
f
b
d
e
f
c
d
e
f
a
d
b
d
c
d
in condition
in condition
in condition
in condition
